commands/issue/update/issue_update.go

```diff
@@ -74,6 +80,8 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 	issueUpdateCmd.Flags().StringP("description", "d", "", "Issue description")
 	issueUpdateCmd.Flags().StringArrayP("label", "l", []string{}, "add labels")
 	issueUpdateCmd.Flags().StringArrayP("unlabel", "u", []string{}, "remove labels")
+	issueUpdateCmd.Flags().BoolP("public", "", false, "Make issue public")
+	issueUpdateCmd.Flags().BoolP("confidential", "", false, "Make issue confidential, overrides --public")
```

Comment:
Add `c` as a shorthand for `confidential`

```suggestion
	issueUpdateCmd.Flags().BoolP("confidential", "c", false, "Make issue confidential, overrides --public")
```